src/main/java/com/xpanxion/java/assignments/student2/Worker2.java

```diff
@@ -1,8 +1,56 @@
 package com.xpanxion.java.assignments.student2;
 
+import com.xpanxion.java.assignments.DataAccess;
+import com.xpanxion.java.assignments.model.Product;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
 public class Worker2 {
 
+    HashMap<Integer, String> depNames = new HashMap<>();
+
     public void ex1() {
```

Comment:
Too many things are being changed in this PR.  The title says ex-3 but 1,2 and 3 are being changed.  I will close this PR.